extensions/binding/org.eclipse.smarthome.binding.dmx/ESH-INF/thing/artnet-bridge.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dmx"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- ArtNet Bridge -->
+    <bridge-type id="artnet-bridge">
+        <label>ArtNet Bridge</label>
+        <description>The ArtNet bridge represents a single DMX universe connected via ArtNet, only unicast</description>
+
+        <channels>
+            <channel id="mute" typeId="mute"/>        
+        </channels>
+
+        <config-description>
+            <parameter name="address" type="text">
```

Comment:
I'll try to explain. What usually is meant if we speak of DMX is DMX512A, a RS485-like serial protocol for transmitting multiple channels. You can have only one sender, but up to 32 receivers (with one or more channels) on one network. ArtNet (or any other DMX over Ethernet like sACN) are similar, they are Point-to-Multipoint protocols, one sender (ESH in this case) and one or more receivers. While the most common application probably will have one receiver in the form of a ArtNet-DMX512A-gateway. ArtNet is only used as a intermediate protocol for transporting the information. But - depending on the use-case - it is possible to have dimmers which directly speak ArtNet or multiple gateways. 
This means that we need to define multiple receivers for each bridge. For sACN it may be possible to use multicast, but for ArtNet we are either limited to one receiver per bridge (which is in my opinion an unnecessary limitation) or allow some sort of address:port combination for each receiver.
I would propose to change the list I have implemented now to a "multiple" but leave address and port combined.